optimization/physical_constants.py

```diff
@@ -16,44 +16,88 @@ def fuel_cond(T):
 #                                                                             #
 #                            Literature Values                                #
 #                                                                             #
 ###############################################################################
-T_fuel_max = 1847.5  # centerline fuel temperature [K]
-T_centerline = T_fuel_max
-k_clad = 108.3  # clad conductivity: W @ 1473 K [W/m-K]
-k_cool = 0.07531  # coolant conductivity [W/m-k]
-# conservative estimate for thermal conductivity at fuel centerline temperature.
-k_fuel = fuel_cond(T_centerline)
-rho_W = 19250  # clad density [kg/m^3]
-rho_UN = 11300  # fuel density [kg/m^3]
-fuel_frac = 0.6  # fraction of fuel in CERMET
+const = {'T_center' : 1847.5, # centerline fuel temp (max) [K]
+         'k_clad' : 108.3,  # clad conductivity: W @ 1473 K [W/m-K]
+         'rho_W' : 19250,  # clad density [kg/m^3]
+         'rho_UN' : 11300,  # fuel density [kg/m^3]
+         'fuel_frac' : 0.6,  # volume fraction of fuel in CERMET
+         }
+# mixed fuel density
+const.update( {'rho_fuel' : const['fuel_frac'] * const['rho_UN'] +
+                           (1 - const['fuel_frac'])*const['rho_W']} )
+# conservative estimate (@centerline T) for fuel thermal conductivity
+const.update( {'k_fuel' : fuel_cond(const['T_center'])} )
 
-###############################################################################
-#                                                                             #
-#                   Calculated Parameters From Power Cycle                    #
-#                                                                             #
-###############################################################################
-dp_allowed = P_in - P_out  # pressure drop limit [Pa]
-Pr = Cp_cool * mu / k_cool  # coolant Prandtl numbe[-]
-# mixed density for CERMET fuel
-rho_fuel = fuel_frac*rho_UN + (1-fuel_frac)*rho_W
+class FlowProperties:
+    """Class to store flow properties and calculate secondary properties from
+    fundamental props.
+    """
+    # default flow properties
+    defaults = {'m_dot' : 0.75, # coolant flow [kg/s]
+                'Q_therm' : 131000, # core thermal power [W]
+                'T' : 1031.45, # bulk coolant temp [K]
+                'P' : 1.766e7, # bulk coolant pressure [Pa]
+                'dp_limit' : 483500, # pressure drop limit [Pa]
+               }
+
+    def __init__(self, **kwargs):
+        """Inialize FlowProperties class and load required flow property data.
+
+        Modified Attributes:
+        -----------------------
+            m_dot: (float) mass flow rate [kg/s]
+            Q_therm: (float) required thermal reactor power [W]
+            T: (float) bulk coolant temperature [K]
+            P: (float) bulk coolant pressure [Pa]
+            dp_limit: (float) power-cycle constrained dp [Pa]
+        """
+        if kwargs:
```

Comment:
This is not robust.  This doesn't restrict the allowable set of `kwargs`.  I recommend using a fixed set of arguments, with defaults for each:
`def __init__(self, m_dot = 0.75, Q_therm=131000, T=1031.45, P=1.766e7, dp_limit = 483500)`

But perhaps you want them to either all be default values or all be specifiedm in which case, you should have an optional dictionary of values that is passed in.